### PR TITLE
automation tests added

### PR DESCRIPTION
**Pull request by Vamshi-Muniganti**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-42

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Comment by kommurunagendra** [comment]
looks good

File: test/e2e/features/step_definitions/steps/loginLogout.steps.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        browser.sleep(SHORT_DELAY);
+        await expect(loginPage.signOutlink.isDisplayed()).to.eventually.be.true;
+        browser.sleep(SHORT_DELAY);
+        await loginPage.signOutlink.click();
+        browser.sleep(SHORT_DELAY);
+    });
+
+
+    Then(/^I should be redirected to EUI dashboard page$/, async function () {
+        // browser.sleep(LONG_DELAY);
```
Comment (fazanki) [inline test/e2e/features/step_definitions/steps/loginLogout.steps.js]: is this still needed

File: test/e2e/support/hooks.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+const jsonReports = `${process.cwd()}/reports/json`;
+const htmlReports = `${process.cwd()}/reports/html`;
+// var xmlReports = process.cwd() + "/reports/xml";
+const targetJson = `${jsonReports}/cucumber_report.json`;
+// var targetXML = xmlReports + "/cucumber_report.xml";
+const { Given, When, Then } = require('cucumber');
+
+
+// defineSupportCode(function({After }) {
```
Comment (fazanki) [inline test/e2e/support/hooks.js]: add todo if you don't want to remove it

File: test/e2e/support/reports.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const reporter = require('cucumberjs-allure-reporter');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+
+const xmlReports = `${process.cwd()}/reports/xml`;
+
+// var createXmlDir = (function() {
+//     if (!fs.existsSync(xmlReports)) {
```
Comment (fazanki) [inline test/e2e/support/reports.js]: remove or add TODO

File: test/e2e/config/conf.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
+const chai            = require('chai');
+const chaiAsPromised  = require('chai-as-promised');
+const minimist        = require('minimist');
+
+chai.use(chaiAsPromised);
+
+const argv = minimist(process.argv.slice(2));
+
+//const specFilesFilter = ['../features/**/*.feature'];
```
Comment (fazanki) [inline test/e2e/config/conf.js]: remove or add TODO

File: test/e2e/config/conf.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  onPrepare() {
+    browser.waitForAngularEnabled(false);
+    global.expect = chai.expect;
+    global.assert = chai.assert;
+    global.should = chai.should;
+  },
+
+  cucumberOpts: {
+    strict: true,
+    // format: ['node_modules/cucumber-pretty'],
```
Comment (fazanki) [inline test/e2e/config/conf.js]: remove or add to do...

File: test/e2e/config/conf.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  }
+];
+
+const cap = (argv.local) ? localConfig : jenkinsConfig;
+
+const config = {
+  framework: 'custom',
+  frameworkPath: require.resolve('protractor-cucumber-framework'),
+  specs: ['../features/**/*.feature'],
+  // specs: [
```
Comment (fazanki) [inline test/e2e/config/conf.js]: remove or add TODO

**Comment by kommurunagendra** [comment]
Good

**Pull request merged**